client/src/app/modules/components/modal/modal.component.ts

```diff
@@ -16,7 +17,7 @@ export class ModalComponent implements OnInit {
 	isModalOpen: boolean = false
 
 	@Input() forceClose: Observable<any> = new Observable()
-	@Input() modalSize: string = 'lg'
```

Comment:
any reason to remove the typing?

Reply:
wasn't compiling for some reason with it